setup/Microsoft.VisualStudio.ProjectSystem.Managed.CommonFiles/Microsoft.VisualStudio.ProjectSystem.Managed.CommonFiles.swixproj

```diff
@@ -0,0 +1,34 @@
+<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project>
+
+  <!-- Note: SwixBuild automatically adds the .vsix to FilesToSign via the AddTargetPathToFilesToSign target. -->
+
+  <Import Project="..\Directory.Build.props" />
+  <!-- This currently loads all SwixBuild packages, and multiple packages existing in your NuGet cache breaks the build. -->
+  <Import Project="$(MicroBuildCoreDir)Microsoft.VisualStudioEng.MicroBuild.Core.props" />
+  <!-- <Import Project="$(SwixBuildDir)Microsoft.VisualStudioEng.MicroBuild.Plugins.SwixBuild.props" /> -->
```

Comment:
You'll see this commented Import (for props/targets) in both the `.swixproj` and `.vsmanproj`. I haven't decided to use this instead of the `MicroBuild.Core` as the Core-based import is how it is *supposed* to work and is more resilient to breaking long-term. However, it will cause **local builds** to be broken that contain multiple *SwixBuild* versions in your NuGet package cache. So, as it currently stands, our team will have issues building locally when this PR is merged until they remove those packages (old versions) from the cache. But long term, the Core-based solution is the right one as it imports *any* MicroBuild plugins that may be required to build these project types. I'm aware that the *SwixBuild* does build them but I'd have to verify if only importing that has all the information (properties/targets) required. Also, other repos I've looked at also do this Core-based import. I also started a thread with the MicroBuild team about this issue with multi-version *SwixBuild* imports using *MicroBuild.Core*, but haven't made much traction there.